src/python/grpcio_tests/tests_aio/unit/call_test.py

```diff
@@ -63,20 +63,15 @@ class TestUnaryUnaryCall(AioTestBase):
             self.assertIs(response, response_retry)
 
     async def test_call_rpc_error(self):
-        async with aio.insecure_channel(self._server_target) as channel:
-            empty_call_with_sleep = channel.unary_unary(
-                "/grpc.testing.TestService/EmptyCall",
+        async with aio.insecure_channel("0.1:80") as channel:
```

Comment:
nit:
1. Please use single quote for strings;
2. Give constants a name to improve readability.